File: 1662-check-if-two-string-arrays-are-equivalent/1662-check-if-two-string-arrays-are-equivalent.java
class Solution {
    public boolean arrayStringsAreEqual(String[] word1, String[] word2) {
        StringBuilder s1 = new StringBuilder();
        for(int i=0;i<word1.length;i++){
            s1.append(word1[i]);
        }
        StringBuilder s2 = new StringBuilder();
        for(int i=0;i<word2.length;i++){
            s2.append(word2[i]);
        }
        
        if(s1.toString().equals(s2.toString())){
            return true;
        }
        return false;
    }
}

File: 1662-check-if-two-string-arrays-are-equivalent/1662-check-if-two-string-arrays-are-equivalent.js
/**
 * @param {string[]} word1
 * @param {string[]} word2
 * @return {boolean}
 */
var arrayStringsAreEqual = function(word1, word2) {
    let ans1 = word1.join("");
    let ans2 = word2.join("");
    return ans1 === ans2;
};